src/components/functions/forms/GenericForm.js

```diff
@@ -10,6 +10,10 @@ const Option = Select.Option;
 const floatPattern = /^\d+\.?\d*$/;
 const intPattern = /^\d+$/;
 
+const filedWidth = 175;
```

Comment:
do you mean `filled` ?

Reply:
sorry miss speling, now it is inputFieldWidth